### DISCUSSION TITLE
Can I use fre:ac to combine MP3 files into M4B audio books?

### DISCUSSION THREAD

**Discussion by halloleo** [enzo1982/freac#373] [Help & Support] [answered]
I have audio book CDs ripped to mp3s and now I would like to combine the mp3s into one large chaptered m4b file.

Can I do this with fre:ac? 

Thanks for pointers.

**Comment by enzo1982** [comment, marked as answer]
Yes, you can do that.

To combine the tracks/files in the joblist into a single output file, just enable the _Encode to a single file_ option at the bottom right of the joblist.

Files created this way will be created with chapters by default. If you disabled chapter generation before, you can enable it under _Tags->Other_ (fre:ac 1.1.6) or _Metadata->Tag fields_ (recent continuous builds) in the configuration dialog. Enable the _Write chapters to files_ option there.

For .m4b audio books, you should also set the file extension to .m4b on the _Format_ tab of the FDK-AAC or Core Audio encoder configuration. This will also mark the file as an audio book in its metadata, so it's better than creating an .m4a file and changing the extension afterwards.

**Reply by halloleo** [reply]
Thanks @enzo1982. I will try this out.

**Comment by GerardEarley** [comment]
I have a realated query to this that is so related it seems silly to create a whole other issue thread.

Is it possible to combine multiple mp3's into a .m4b file (or mkv for that matter) without also transcoding them?

I've lots of audiobooks made up from multiple mp3 that are already at quite a low bitrate (<=64kbps) transcoding them would only reduce their quality further.

**Reply by enzo1982** [reply]
That's not possible with fre:ac.

And while it would be possible in theory, I doubt that you would have much success with this in practice. .m4b is basically just an MP4 container that can contain different types of audio. Most often that's AAC or ALAC. There also exists a way to store MP3 data in MP4 container files. However, that combination is very rarely used in practice. So rarely, that I have not come across an MP4/MP3 file yet and do not know any software that can create those. Not even FFmpeg supports creating MP4 files with MP3 audio content.

And even if you found a software that can create .m4b files with MP3 content, you would hardly find a player that can play such files. So while possible in theory, in practice it's basically impossible to achieve this.

**Reply by GerardEarley** [reply]
Many thanks for the timely reply and I understand your point completely.

I'm really just trying to concatenate all these audiobook mp3's into one file with chapter markers to make listening to them easier on my phone. I usually play audiobooks at 120-130% speed and needing to reset the speed every time it changes file is really annoying.

But the chjapter markers are a nice-to-have rather than an essential.

Thanks anyway 👍
